### PR TITLE
Main Curriculum Youtube Project

### PR DESCRIPTION
**Pull request by emiliocm9**
Create a web page simulating a Youtube Front-End page. 
Gabriel Silveira & Emilio Contreras worked on this project for Microverse Full Time Students.

### CODE REVIEW

**Review by MarioDena — CHANGES_REQUESTED** [review]
Hello team good job on your project 👍  
However there are still some changes that need to be made:

## Changes required 

- [ ] There are some repeated rules on your CSS, remember to keep your code DRY. You can read this article for reference: https://speckyboy.com/good-bad-css-practices/

- [ ] Please don't used ID's for styling this is considered a bad practice use classes instead.

- [ ] Remember to validate your HTML using this: https://validator.w3.org/#validate_by_upload  and fix the errors.

## Optional Changes

- [ ] The general layout looks similar but kind of off. Try to play a game of find the differences between your project and the original you-tube page. Maybe add more videos to the side bar, etc... This changes is [OPTIONAL].

- [ ] Specify the resolution your project looks better in. Right now I'm on 1080p and the video for example looks way off. You are not yet required to make responsive projects and this will help future TSEs to test your project on the same resolution your worked on. This changes is [OPTIONAL].

After making the requested changes please resubmit your project for review

Cheers! 😄 

## Code review by Mario Enrique Delgado Naranjo

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Youtube-VideoPlayerPage
+The goal here isn’t to produce an exact replica of the YouTube video page, but to focus on making the media elements show up. That means two things – embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side.
+
+![Youtube Project made by Gabriel & Emilio](Images/ProjectSS.png?raw=true "Youtube Project made by Gabriel & Emilio")
+
```
Comment (majovanilla) [inline README.md:5]: - [ ] Great description!!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+It shows the video title.
+It shows the number of views.
+It has like and dislike icons.
+It has a share button.
+It shows the avatar of the user who uploaded the video.
+It has a subscribe button.
+Sidebar Recommended videos
+It shows the recommended video’s thumbnail preview (as image not as an actual video).
+It shows the title of the video.
+It shows the number of views.
```
Comment (majovanilla) [inline README.md]: These are not prerequisites. This section refers to when someone wants to clone your project and they need to have something installed before to be able to run it. You could change the title to Project details.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey everyone!

## Good job on this project! 👍

- Good Readme file
- Good Pull Request Message
- Good job making the previously required changes.

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your stickler errors. There are several here, please make the changes.

- Check the comments in your code below this message.

**You are almost done!! Your project was very good and you just missed the linters!**

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi **TEAM**, you have done a great job implementing the changes pointed out by the previous TSE's 👍 
Thanks to them and your hard work the project looks even better 🙂 I hereby, approve of your project, go ahead and merge to master and submit project completion form.

Good luck with your next project 👍 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo

**Pull request merged**